transformer_engine/pytorch/attention.py

```diff
@@ -3404,6 +3407,7 @@ def __init__(
         cp_group: Optional[dist_group_type] = None,
         cp_global_ranks: List[int] = None,
         cp_stream: torch.cuda.Stream = None,
+        norm_factor: Optional[float] = None,
```

Comment:
I don't think this is a good name since it doesn't involve norms. `softmax_scale` (from [`flash_attn.flash_attn_qkvpacked_func`](https://github.com/Dao-AILab/flash-attention/blob/320fb59487658f033f56711efd3d61b7c7a6f8f3/flash_attn/flash_attn_interface.py#L642) or `scale` (from [`torch.nn.functional.scaled_dot_product_attention`](https://pytorch.org/docs/stable/generated/torch.nn.functional.scaled_dot_product_attention.html#torch-nn-functional-scaled-dot-product-attention)) is more descriptive:
```suggestion
        softmax_scale: Optional[float] = None,
```

Reply:
I have changed the name of this kwarg